pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -478,8 +477,20 @@ class Build {
                                     }
                                 }
                                 context.println "Use Test_Job_Auto_Gen to generate AQA test job with parameters: ${updatedParams}"
-                                context.catchError {
-                                    context.build job: 'Test_Job_Auto_Gen', propagate: false, parameters: updatedParams
+                                int x = 3
+                                while ( x > 0 ) {
+                                    x--
+                                    def autogen_result = "BLANK"
+                                    context.catchError {
+                                        def testJob = context.build job: 'Test_Job_Auto_Gen', propagate: false, parameters: updatedParams
+                                        autogen_result = testJob.getResult()
+                                    }
+                                    if ( !autogen_result.equals("SUCCESS") && x > 0 ) {
+                                        context.println "This script will now pause for 5 minutes before retrying."
+                                        sleep(5*60*1000)
```

Comment:
recommend extracting that to a value and using that var in the sleep and the echo